src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(result, number) {
+      return result + number;
+    },
+
+    subtract(result, number) {
+      return result - number;
+    },
+
+    multiply(result, number) {
+      return result * number;
+    },
+
+    divide(result, number) {
+      return result / number;
+    },
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
```

Comment:
let's improve your code, I think we can avoid putting `this.result` directly in our callback.

we can use `bind`, `call` or `apply`  to have proper `this` inside our callback.



[call-apply-and-bind](https://www.freecodecamp.org/news/understand-call-apply-and-bind-in-javascript-with-examples/)